usage/importing-history.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Importing your History
+parent: Usage
+permalink: /usage/importing-history/
+---
+
+FreeTube has the ability to import your history from a variety of services. Here you will find instructions on how to import from each service.
+
+## From YouTube
+
+1. Login to your YouTube account
+2. Click on your profile picture in the top right corner of the web page
+3. Click on "Your data in YouTube" in the displayed drop down
+4. Click on "More" in the "Your YouTube dashboard" card
+5. Click on "Download YouTube data"
+6. Click on "All YouTube data included" and deselect everything except for "History"
+7. Click "Next step"
+8. Change the History file format to "JSON"
+9. Click on "OK"
+10. Click "Next step"
+11. Select your preferred method of deliver (Email, Dropbox, etc.) and click on "Create Export" (Instructions from here on out will assume that you selected email)
+12. Check your email and download the .zip file that is provided.
+13. Extract the .zip file
+14. The file you want will be in `Takeout/YouTube and YouTube Music/history/history.json`
+
+Once you have this file, you can import this file into FreeTube by going to `Settings->Data Settings->Import History`
+
+## From Invidious
+
+While logged into your Invidious account, go to the settings page. Scroll down to the bottom of the page and look for the "Import/Export Data" link. On the new page, look at the bottom for "Import Subscriptions as OPML (for NewPipe and FreeTube)".
+
+Once you have this file, you can import this file into FreeTube by going to `Settings->Data Settings->Import Subscriptions`
+
+## From NewPipe
+
+Go to your subscriptions page (Not your settings page!). On this page, click on the `Subscriptions` title to open up a drop down. In this drop down,click on the "Export to File" button. Transfer this file to your PC running FreeTube.
+
+Once you have this file, you can import this file into FreeTube by going to `Settings->Data Settings->Import Subscriptions`
+
+## From other FreeTube instances
+
+Go to the Settings page of the FreeTube app. Under "Data Settings", click on "Export History".
+
+Once you have transferred this file to your new machine, you can import this file into FreeTube by going to `Settings->Data Settings->Import History`
+
+## From Somewhere Else
+
+At this time, we do not support importing history from anywhere other than the above methods. Please ask the maintainer of your app if they would be open to supporting exporting their history into one of the above formats and then use that import function within FreeTube to import your history.
+
+## Troubleshooting Issues
```

Comment:
havent changed this yet

Reply:
> Browse your exported file and check for any banned / removed channels. Exporting from YouTube will still include these channels in your exported file, however FreeTube will obviously be unable to import the

Is this applicable?

Reply:
No...
Or more like the data is too massive for human to check
This is importing history not channels